### ISSUE TITLE
Optimise data processing for output of “OK”

### ISSUE THREAD

**Issue by elfring** [traviscross/mtr#454]
The printing for the text “`OK`” was specified in four statements of [the function “csv_close”](https://github.com/traviscross/mtr/blob/0886eee5bab164fee50f94d542de71fca8a2dc68/ui/report.c#L509 "Update candidate") together with a format string parameter “`%s`”.
💭 I suggest to use the literal within the format string directly instead.

**Comment by rewolff** [comment]
No. There is a list-of-stuff-between-commas and I think it is easier for us humans to make an error in matching the place on the output with one of the items being "different". 
```
    1,2,3,4,5
```
in the output corresponds to 
```
  sprintf (...
    1, 
    2, 
    3, 
    4, 
    5, 
``` 
in the source code.  even if say "3" happens to be the constant "OK"
